pages/implementation/reference/rescript.mdx

```diff
@@ -156,12 +157,12 @@ Every event you define in your tracking plan in Avo gets a function named after
 
 ##### Arguments
 
-`eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array, object and any.
+`eventProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any.
 Every event property attached to the event in the Avo UI gets a corresponding argument.
 The argument key is camelCase version of the property name defined in the Avo UI.
 Pass the value of the property to track here.
 
-`userProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array, object and any.
+`userProperty`: type defined in the Avo tracking plan, can be string, int, long, float, bool, array and any.
```

Comment:
Does the same as @bragur  mentioned apply here?
https://github.com/avohq/docs/pull/212/files#r681673826